### PR TITLE
Django sass process 622

### PR DESCRIPTION
**Pull request by groovecoder**
I'm happy to wait on this, but I wanted to put it up here for consideration ...

This includes https://github.com/codesy/codesy/pull/625 and adds `django-sass-processor` so we can just keep `.scss` files in the repo, and let `django-sass-processor` compile the `.css` files during deployment release.

The alternative to this is either:

1. Check the compiled `.css` files into the repo and hope we don't forget them each time
2. Use Heroku Buildpack Multi to compile the `.scss` files into `.css` with Ruby `sass` command

### CODE REVIEW

File: codesy/templates/base.html [hunk 1]
```diff
@@ -11,7 +12,7 @@
         <link rel="shortcut icon" href="{% static "img/favicon.png" %}">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/foundation/6.2.4/foundation.min.css">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/foundation-icons/3.0/foundation-icons.min.css">
-        <link rel="stylesheet" href="{% static "css/codesy-home.css" %}">
+        <link rel="stylesheet" href="{% sass_src "css/scss/codesy-home.scss" %}">
```
Comment (groovecoder) [inline codesy/templates/base.html]: `sass_src` automatically converts this to look for the compiled `.css` file.

**Review by maxxcrawford — COMMENTED** [review]
What is the development process when you're working on local changes with the Django tool? Do we still leave the README the same for now, or reference the [offline compile option](https://github.com/jrief/django-sass-processor#offline-compilation) from the Django-sass-processor documentation?

**Comment by groovecoder** [comment]
As I understand it, `django-sass-processor` automatically compiles the `.sass` into the `.css` every time it's included in a template. So the local development process is just to save the `.sass` file and reload the page.

This PR also adds a new `release-tasks.sh` script for heroku to run `python manage.py compilescss` after it runs the db migrations. So during the stage & production deploys, we compile the sass "offline".

**Comment by maxxcrawford** [comment]
10-4. This is good by me!

**Pull request closed without merging**